frontend/packages/dev-console/src/components/monitoring/alerts/MonitoringAlerts.tsx

```diff
@@ -123,10 +175,10 @@ export const MonitoringAlerts: React.FC<Props> = ({ match, rules, filters, listS
   );
 
   const Content = React.useMemo(() => {
-    if (loading && !loadError) {
+    if (!alerts?.loaded && !alerts?.loadError) {
```

Comment:
Do we also need to handle the case where the alerts are loaded, but the rules are pending here?

Reply:
Alerts are extracted from rules so they are loaded at the same time